services/core-api/app/api/mines/permits/permit/models/permit.py

```diff
@@ -88,19 +88,17 @@ def validate_permit_no(self, key, permit_no):
         return permit_no
 
 
-from marshmallow_sqlalchemy import ModelConversionError, ModelSchema, ModelConverter, fields
+# from marshmallow_sqlalchemy import ModelConversionError, ModelSchema, ModelConverter, fields
```

Comment:
Is this change intentional? @Jsyro

Reply:
Yeah, commenting it out was intentional on my part. 

From my understanding, this was half working code, everything should be updated/working in his PR.  (just looked now and saw that his PR was merged, so I'll make sure to chat with him tomorrow)

Reply:
removed